quupe-RN/js/redux/modules/Token.js

```diff
@@ -53,7 +53,7 @@ export const deleteUserToken = token => dispatch => {
 };
 
 const initialState = {
-    token: '',
+    token: queryToken(),
```

Comment:
Please make sure that this query will not throw an error, it should either be null or return a token.